index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <title>Title</title>
+</head>
+<body>
+<header class="outer-container">
```

Comment:
Enorm groot compliment van mij, je structuur en opbouw van je HTML zowel als het gebruik van de tags en semantiek is echt super goed opgezet! Echt drie stickers!!